indico/modules/events/templates/header.html

```diff
@@ -32,7 +34,9 @@
            title="{% trans %}Newest event{% endtrans %}"></a>
     {% endif %}
 
-    <span class="separator"></span>
+    {% if event.category or ( (rel['first'], rel['prev'], rel['next'], rel['last'])|any ) %}
```

Comment:
I think `x or y or z or ...` would be more readable ;) (but what I certainly don't want is a space after an opening or before a closing parenthesis ;))

Jinja also lets you shortcut the dict item access via `rel.first` etc

Reply:
Indeed, that was overly complicated. Also I didn't know about the `rel.first`!

Reply:
Basically what Jinja  does is catching AttributeError/KeyError when accessing with `.`/`[]` and then trying the other way of accessing. So while it's slightly more performant to use the "right" way, that's negligible and the dot notation is more readable for sure.

One useful pitfall to keep in mind though: `foo.items` for a dict with an `items` key won't work because dicts already have an attribute with that name!